includes/definitions/discovery/gaia.yaml

```diff
@@ -78,6 +79,26 @@ modules:
                         - { value: 0, descr: Up , graph: 2, generic: 0 }
                         - { value: 1, descr: Attention, graph: 1, generic: 3 }
                         - { value: 2, descr: Down, graph: 0, generic: 1 }
+                -
+                    oid: mglsGWState
```

Comment:
Missing test data for this

Reply:
Test data for `mglsGWState` included. Tks.